plugins/obs-ffmpeg/ffmpeg-mux/ffmpeg-mux.c

```diff
@@ -257,6 +302,16 @@ static bool init_params(int *argc, char ***argv, struct main_params *params,
 
 	*p_audio = audio;
 
+	get_opt_str(argc, argv, &global_stream_key, "stream key");
+	if (strcmp(global_stream_key, "") != 0) {
+		struct dstr tmp = {0};
+		dstr_copy(&tmp, params->file);
+		dstr_replace(&tmp, global_stream_key, "{stream_key}");
+		strcpy(params->file, tmp.array);
```

Comment:
Shouldn't tmp.array be assigned to printable_file?

Reply:
Yes! I think this occurred when I changed the names--you're completely right. Even though the log looks fine it doesn't successfully stream to YouTube. Thanks for the catch.